src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/DefaultLSPRequestInvoker.cs

```diff
@@ -178,13 +181,16 @@ internal class DefaultLSPRequestInvoker : LSPRequestInvoker
             [EnumeratorCancellation] CancellationToken cancellationToken)
         {
             var serializedParams = JToken.FromObject(parameters);
-            Func<ITextSnapshot, JToken> parameterFactory = (_) => serializedParams;
+            JToken ParameterFactory(ITextSnapshot _)
```

Comment:
I'm curious about the reasoning for this change?

Reply:
`IDE0039: Use local function` I don't actually have strong feelings about this one, so I'm fine if we change the expectation. What do people think?

Reply:
TIL.. fair enough then